torch_geometric/loader/dataloader.py

```diff
@@ -1,5 +1,5 @@
-from collections.abc import Mapping, Sequence
-from typing import List, Optional, Union
+from collections.abc import Mapping
+from typing import List, Optional, Sequence, Union
```

Comment:
We need to use `typing.Sequence` instead of `collections.abc.Sequence` for compatibility with Python<=3.8.